docs/the-new-architecture/landing-page.md

```diff
@@ -3,11 +3,26 @@ id: landing-page
 title: Introduction
 ---
 
-This section is the entry point for the new guide’s documentation.
-It contains some basic information about the New Architecture: its pillars, the version from which it has been made available and other very high-level information. Then, it should present the guide’s structure itself.
+Starting in version 0.68, React Native provides the New Architecture, which offers developers new capabilities for building highly performant and responsive apps. Visit [Why a New Architecture](why) to learn more about what drove the decision to re-architect, and the benefits it provides.
 
-This section should also contains a sort of Driving guide based on the user use-cases:
+The core concepts are explained by the [Pillars of the New Architecture](pillars):
```

Comment:
I would rephrase this slightly. It looks a little bit abrupt moving from the `why` to the `components` like that.

We could add a sentence on the lines of:
> In order to achieve these benefits, we had to rethink how Native Modules and Native Components works. This led us to develops the [Pillars of the New Architecture](pillars):

What do you think?

Reply:
Yeah, good idea!